src/PrebidMobile/PrebidMobile/AdUnits/Utils.swift

```diff
@@ -241,6 +241,63 @@ public class Utils: NSObject {
         return gcSize
     }
 
+    func getObjectWithoutEmptyValues(_ dict: [AnyHashable: Any]) -> [AnyHashable: Any]? {
```

Comment:
The utils is becoming a heavy lifter. Do you want to separate out the features?

Reply:
I will update the PR